site-src/contributing/index.md

```diff
@@ -0,0 +1,108 @@
+# How to Get Involved
+
+This page contains links to all of the meeting notes, design docs and related discussions around the different APIs managed by the Multicluster SIG.
+
+## Feedback and Questions
+
+For general feedback, questions or to share ideas please feel free to [create a
+new discussion][gh-disc].
+[gh-disc]:https://github.com/kubernetes-sigs/gateway-api/discussions/new
```

Comment:
```suggestion
For general feedback, questions or to share ideas please feel free to [create a
new discussion against the site repo][gh-disc].
[gh-disc]:https://github.com/lauralorenz/sig-multicluster-site-proposal/discussions/new
```

Reply:
fixed in [cbb076d](https://github.com/lauralorenz/sig-multicluster-site-proposal/pull/1/commits/cbb076d91bde167425c76cdbbc42a300e8c7d65a)